### PR TITLE
Implement DB Schema

### PR DESCRIPTION
**Pull request by ksmolko**
Created an initialization script for the PostgreSQL DB.

### CODE REVIEW

File: server/sql/init_db.sql [hunk 1]
```diff
@@ -0,0 +1,399 @@
...
+-- Only needs to be run once per server
+
+CREATE DATABASE dnd_db;
+
+\connect dnd_db
+
+CREATE TABLE Player (
+    username        text PRIMARY KEY,
+    password        text,
+    email           text,
```
Comment (knsjerome) [inline server/sql/init_db.sql:11]: `email` should have a constraint that makes it unique.

**Review by knsjerome — COMMENTED** [review]
Other constraints can be added (although not strictly necessary):

- `NOT NULL` constraints on certain foreign keys
- `CHECK >= 0` on attributes that should never be negative
etc

File: server/sql/init_db.sql (hunk 1 above)
Comment (ksmolko) [reply server/sql/init_db.sql]: Good note. In my new commit, I've made any FKs that aren't also primary keys `NOT NULL`, added `DELETE`/`UPDATE` conditions to each FK, added check constraints to most numeric attributes, and made email unique.

**Review by knsjerome — APPROVED** [review]

File: server/sql/init_db.sql (hunk 1 above)
Comment (knsjerome) [reply server/sql/init_db.sql]: Looks good - approved. I'm sure if anything needs to be changed it'll come up during our experimentation.

**Pull request merged**